moduleroot/.travis.yml.erb

```diff
@@ -29,6 +28,8 @@ matrix:
 <% end -%>
     - rvm: 2.4.1
       env: PUPPET_VERSION=5.0
+    - rvm: 2.5.1
```

Comment:
Should this be hidden behind `@configs['test_all_rvms']` so if it's false we still only with the Puppet bundled versions?

Reply:
changed.